src/datagrid/DataGrid.tsx

```diff
@@ -953,7 +951,7 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
             >
                 {footer && footer.hideShowColBtn && this.buildHideShowColumnsBtn()}
                 <div className={ClassNames.DATAGRID_FOOTER_DESC}>
-                {pagination && pagination ? this.buildDataGridPagination(): this.buildFooterContent() }
+                    {pagination && pagination ? this.buildDataGridPagination() : this.buildFooterContent()}
```

Comment:
double pagination check

Reply:
@alexander-rassanov  thanks for noticing this. @ben-schumacher  can you pleas take care of this DataGrid code as well ?